File: 2-Semestre/Taller de Programación/Personal/Python-C#/Ejercicio 6.cs
/*
Transforme pesos (CPL) a dólares (USD), imprimiendo el resultado en pantalla.
La cantidad de pesos (CLP) es ingresada por el usuario.
*/

using System;

namespace Ejercicio_6
{
    public class Program
    {
        public static void Main(string[] args)
        {
            double usd, clp;

            do
            {
                Console.Write("Ingrese el valor del dólar actual: ");
                usd = Convert.ToDouble(Console.ReadLine());

                if (usd <= 0)
                {
                    Console.WriteLine("[!] El valor del dólar no puede ser menor o igual a 0");
                }
            } while (usd <= 0);

            do
            {
                Console.Write("Ingrese la cantidad de pesos (CLP): ");
                clp = Convert.ToDouble(Console.ReadLine());

                if (clp <= 0)
                {
                    Console.WriteLine("[!] La cantidad de pesos Chilenos no puede ser menor o igual a 0");
                }
            } while (clp <= 0);

            double conversión = clp / usd;
            Console.WriteLine($"\n${clp} pesos Chilenos a dólares son {conversión}");
        }
    }
}

File: 1-Semestre/Desarrollo de algoritmos/Actividades/Diagramas de flujo-Python (Semana 5)/Ejercicio 6.py
'''
Transforme pesos (CPL) a dólares (USD), imprimiendo el resultado en pantalla.
La cantidad de pesos (CLP) es ingresada por el usuario.
'''

usd = float(input("Ingrese el valor del dólar actual: "))
clp = int(input("Ingrese la cantidad de pesos (CLP): "))
conversion = 0

while usd <= 0 or clp <= 0:
    print("[!] El valor del dólar y la cantidad de pesos deben ser mayores a 0.")
    usd = float(input("Ingrese el valor del dólar actual: "))
    clp = int(input("Ingrese la cantidad de pesos (CLP): "))

conversion = clp / usd
print(f"${clp} pesos a dólares son {conversion:.2f}") ## .2f para redondear a 2 decimales